src/submission/models.py

```diff
@@ -1045,6 +1045,13 @@ def production_assignment_or_none(self):
         except ObjectDoesNotExist:
             return None
 
+    @property
+    def citations(self):
+        for citation in self.forwardlink_set.all():
+            print(citation.pk)
+
+        return self.forwardlink_set.all()
```

Comment:
This is a Question: Does this actually work? I'd have thought you'd need to build a queryset of `BookLink` and `ArticleLink`, since I don't think querysets can mixed types